content/en/tracing/setup_overview/compatibility_requirements/dotnet-framework.md

```diff
@@ -15,51 +15,82 @@ further_reading:
       tag: 'GitHub'
       text: 'Examples of Custom Instrumentation'
 ---
+<div class="alert alert-warning"> 
+  <strong>Notes:</strong><br><ul><li>Datadog automatic instrumentation relies on the .NET CLR Profiling API. This API allows only one subscriber (for example, APM). To ensure maximum visibility, run only one APM solution within your application environment.</li><li> If you are using both automatic and custom instrumentation, it is important to keep the package versions (for example, MSI and NuGet) in sync.</li></ul>
+</div>
 
 ## Compatibility
+Datadog .NET application performance monitoring supports a robust set of features to provide critical information about .NET applications & services: 
 
 - The .NET Tracer supports all .NET-based languages (for example, C#, F#, Visual Basic).
 
-- The .NET Tracer supports instrumentation on .NET Framework 4.5 and above (using CLR v4.0). It also supports [.NET Core][1].
+- Monitor [runtime metrics][1] to help troubleshoot bugs and detect resource inefficiencies in your application.
 
-- The .NET Tracer library for Datadog is open-source. For more information see the [tracer Github repository][2].
+- Ability to correlate [traces & application logs][2].
 
-<div class="alert alert-warning"> 
-  <strong>Notes:</strong><br><ul><li>Datadog automatic instrumentation relies on the .NET CLR Profiling API. This API allows only one subscriber (for example, APM). To ensure maximum visibility, run only one APM solution within your application environment.</li><li> If you are using both automatic and custom instrumentation, it is important to keep the package versions (for example, MSI and NuGet) in sync.</li></ul>
+- The .NET Tracer library for Datadog is open-source. For more information, see the [tracer Github repository][3].
 
-</div>
+### .NET runtimes
+The .NET Tracer supports instrumentation for [.NET Core][4] & NET Framework.
 
+| Supported .NET Framwork Versions | Vendor End of Life |
```

Comment:
On this table you went from oldest to newest, but the .NET Core you went to newest to oldest. We should standardize on one order. I personally prefer the other approach so at the top of the table you see the latest versions of the runtime.

Reply:
Good catch thanks! Newer up top - works for me. I'll make the adjustments later.

Reply:
Updated the .NET Framework from newer on top to older at bottom. Changes are [here](https://github.com/DataDog/documentation/pull/11200/commits/d4260a023194b2e0791277b5c9242b0903fff21e).

Marking as resolved.